domain/model/modelmigration/export_test.go

```diff
@@ -0,0 +1,50 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package modelmigration
+
+import (
+	"context"
+
+	"github.com/juju/description/v8"
+	gomock "go.uber.org/mock/gomock"
+	gc "gopkg.in/check.v1"
+
+	coremodel "github.com/juju/juju/core/model"
+	"github.com/juju/juju/internal/uuid"
+)
+
+type exportSuite struct {
+	modelExportService *MockExportService
+}
+
+var _ = gc.Suite(&exportSuite{})
+
+func (e *exportSuite) setupMocks(c *gc.C) *gomock.Controller {
+	ctrl := gomock.NewController(c)
+
+	e.modelExportService = NewMockExportService(ctrl)
+
+	return ctrl
+}
+
+func (e *exportSuite) TestModelEnvironVersionExport(c *gc.C) {
+	defer e.setupMocks(c).Finish()
+
+	newUUID := uuid.MustNewUUID().String()
```

Comment:
There is a package `core/model/testing` with a func `GenModelUUID` that should be used instead of this.

Reply:
Noted with thanks tom!